yt_dlp/extractor/cinetecamilano.py

```diff
@@ -0,0 +1,63 @@
+from json import JSONDecodeError
+from urllib.error import HTTPError
```

Comment:
```suggestion
import json
import urllib.error
```
make necessary changes below

Reply:
I'm not sure about this one, should I import `json` and then catch `json.JSONDecodeError` ?

Reply:
yes